packages/devextreme/testing/tests/DevExpress.ui.widgets/treeView.expanded.tests.js

```diff
@@ -545,6 +545,45 @@ module('Expanded items', {
         assert.ok(nodeElements.eq(2).hasClass(TREEVIEW_NODE_CONTAINER_OPENED_CLASS), 'item 111');
     });
 
+    QUnit.test('onItemCollapsed event should be rised when collapseItem called after expandAll(T1202248)', function(assert) {
+        const itemCollapsedSpy = sinon.spy();
+        const done = assert.async();
+        assert.expect(1);
```

Comment:
redundant assert?

Reply:
it's not a assert. [it's expect](https://api.qunitjs.com/assert/expect/) function call.